test/test_stablehlo_dump.py

```diff
@@ -24,11 +26,40 @@ def test_resnet18(self):
     xla_resnet18 = xla_resnet18.to(device)
     data = torch.randn(4, 3, 224, 224, device=device)
     output = xla_resnet18(data)
-    stablehlo = xm.get_stablehlo([output])
+    stablehlo = xm.get_stablehlo([output]).decode('utf-8')
     self.assertEqual(stablehlo.count("stablehlo.convolution"), 20)
-    stablehlo = xm.get_stablehlo()
+    stablehlo = xm.get_stablehlo().decode('utf-8')
     self.assertEqual(stablehlo.count("stablehlo.convolution"), 20)
 
+  def test_resnet18_inference(self):
```

Comment:
let's keep this test as a `stablehlo_dump` test, and put inference/saved_model spcified bits to a different tests. I was talking to @lsy323 about creating a new dir called `stablehlo` or `inference` under `/test/`